### DISCUSSION TITLE
Multiple Full nodes

### DISCUSSION THREAD

**Discussion by GeorgePapageorgakis** [Chia-Network/chia-blockchain#6017] [Support]
I know that it is not recommended but, should it cause any problems when you have multiple nodes farming on the same public key?

**Comment by tiny0little** [comment]
this should be no problem, as long as you can make sure inbound `tcp8444` works for them. if they are on the same network, you might get into trouble...

**Reply by GeorgePapageorgakis** [reply]
What could happen? They are on the same subnet.

**Reply by tiny0little** [reply]
only one will work, the one which has working inbound `tcp8444`

**Reply by GeorgePapageorgakis** [reply]
How can I make sure that chia connections are working? I guess not seeing any errors in the debug logs? Opening ports is a serious security risk...

**Comment by joelrb** [comment]
Read:
https://github.com/Chia-Network/chia-blockchain/wiki/FAQ#why-should-i-not-run-more-than-one-node-on-a-home-network-and-whats-this-about-upnp

**Reply by GeorgePapageorgakis** [reply]
yes I have disabled upnp for the 2 nodes and i have left only 1 with it.
However it is disabled in my modem since I know its a security risk to leave it open. Could this be a problem?

**Reply by joelrb** [reply]
> yes I have disabled upnp for the 2 nodes and i have left only 1 with it.
> However it is disabled in my modem since I know its a security risk to leave it open. Could this be a problem?

When you disable UPnP, your router will no longer respond to any incoming connection requests. So, you have to manually open ports (8444 for Chia) and forward request to your Chia machine local network IP.

**Reply by GeorgePapageorgakis** [reply]
So its mandatory for incoming connections only or do we need to have for outgoing connections too? How can I see if this is the problem? I dont  see any error regarding 8444 port.

**Reply by joelrb** [reply]
> So its mandatory for incoming connections only or do we need to have for outgoing connections too? How can I see if this is the problem? I dont see any error regarding 8444 port.

Read, if you haven't yet.
https://github.com/Chia-Network/chia-blockchain/wiki/Resolving-Sync-Issues---Port-8444

**Reply by GeorgePapageorgakis** [reply]
I have read it but that doesnt make port forwarding safer. You need to know how to port forward safely, creating multiple subnets or using 2 routers so that you can avoid any malicious intruders in your systems. Am I missing something here?

**Reply by joelrb** [reply]
> I have read it but that doesnt make port forwarding safer. You need to know how to port forward safely, creating multiple subnets or using 2 routers so that you can avoid any malicious intruders in your systems. Am I missing something here?

Opening ports was discussed in the security briefing held May 29. Here's the youtube link: https://www.youtube.com/watch?v=n0P9FO1DafM

Just forward video to 43:35. Although Justin talks only about opening ports, it's worth listening to.

**Comment by BrandtH22** [comment]
@GeorgePapageorgakis >, since this ticket was opened a number of years ago without further comments it seems this issue has been resolved. 
We will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions or comments.

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
